xl-objects-multipart.go

```diff
@@ -245,6 +250,10 @@ func (xl xlObjects) CompleteMultipartUpload(bucket string, object string, upload
 		return "", toObjectErr(err, minioMetaBucket, uploadIDIncompletePath)
 	}
 
+	// Hold write lock on the destination before rename
+	nsMutex.Lock(bucket, pathJoin(bucket, object))
```

Comment:
Isn't the destination already "bucket, object" shouldn't this be 

nsMutex.Lock(bucket, object) ?